app/models/solidus_paypal_braintree/source.rb

```diff
@@ -1,12 +1,13 @@
 module SolidusPaypalBraintree
-  class Source < ApplicationRecord
+  class Source < Spree::PaymentSource
```

Comment:
Can you change this to `SolidusSupport.payment_source_parent_class` please?
See https://github.com/solidusio/solidus_support/pull/1